internal/script/payout.go

```diff
@@ -36,7 +36,9 @@ func ExecutePayout(
 		return nil, fmt.Errorf("unable to fetch latest metadata, because of %v", err)
 	}
 
-	keyringPair, err := signature.KeyringPairFromSecret(privateKey, "")
+	// Use wildcard 42 - Generic Substrate wildcard
+	// https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)#address-type
+	keyringPair, err := signature.KeyringPairFromSecret(privateKey, 42)
```

Comment:
@mpetrun5 @mpetrunic I think it is ok to use this 42 wildcard/generic substrate network identifier here.

For more details you can read [this](https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58)#address-type)

Reply:
extract to constant :)

Reply:
:+1: